.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] It is required that test.md file should be created and ignore it by adding it to this file. Kindly create the file and it to this file in order to ignre it from appearing on your repository.


### Project Requirement
![test-file](https://user-images.githubusercontent.com/26597432/235256611-6b873794-93bb-42a9-9b04-ce27a7424293.png)